src/main/java/net/fabricmc/loom/LoomRepositoryPlugin.java

```diff
@@ -68,6 +68,18 @@ private void declareRepositories(RepositoryHandler repositories, LoomFiles files
 			repo.setUrl(MirrorUtil.getFabricRepository(target));
 		});
 
+		repositories.exclusiveContent(exclusiveContent -> {
+			exclusiveContent.forRepositories(repositories.maven(repo -> {
+				repo.setName("FabricExclusive");
+				repo.setUrl(MirrorUtil.getFabricRepository(target));
+			}));
+
+			exclusiveContent.filter(filter -> {
+				filter.includeGroup("net.fabricmc");
+				filter.includeGroupByRegex("net.fabricmc.*");
```

Comment:
Is the first `includeGroup` even needed if you have this? (`net.fabricmc` definitely matches the regex `net.fabricmc.*`)

Reply:
Ill check.